docs/reST/themes/classic/static/pygame.css_t

```diff
@@ -689,3 +669,35 @@ pre.commentContent {
     border: 0;
     white-space: pre-wrap;
 }
+
+/* -- responsive design --------------------------------------------------------- */
+
+@media only screen and (max-width: 680px) {
+    /* Make tables be responsive-ish. */
+    table, thead, tbody, th, td, tr {
+        display: block;
+    }
+
+    div.body img.inlined-right {
+        float: none;
+        display: block;
+        margin: auto
+    }
+
+    span.linenos {
+        margin-right: 10px;
+    }
+
+    .addcomment input, a.commentButton {
+        font-size: 10px;
+    }
+
+    pre {
+        white-space: pre-wrap;
+        word-wrap: break-word;
+    }
+
+    .toc td {
+        width: 100%;
+    }
+}
```

Comment:
Super minor, but I believe it's conventional to have files end with newlines. (Something I learned in my camera PR).

Unix systems might not interpret this correctly.

Just go onto the next line, give it 0 content, and save.